server/src/main/java/org/elasticsearch/cluster/routing/allocation/ShardsAvailabilityHealthIndicatorService.java

```diff
@@ -264,6 +499,27 @@ public List<HealthIndicatorImpact> getImpacts() {
             return impacts;
         }
 
+        public List<UserAction> getUserActions(boolean includeDetails) {
```

Comment:
Can you add javadoc for this method?

Reply:
Added in [1cdba6f](https://github.com/elastic/elasticsearch/pull/85071/commits/1cdba6f11ed80f91d470a1631a73343262cd9bf5)